docs/architecture.md

```diff
@@ -1,103 +1,170 @@
 # OpenVoiceOS Architecture
-**Editors Note**
-Most of the text in the entire architecture section will not remain, it's my notes so I can create some architecture drawings and simplified information.
+**This section can be a bit technical, but is included for refrence**
 
-## Bus Service
+OVOS is a collection of modular services that work together to provide a seamless, private, open source voice assistant.  Every OVOS service runs as an indivudually and interacts with one another.
+
+The suggested way to start OVOS is with systemd service files.  Most of the supported images run these services as a normal `user` instead of system wide.  If you get and error when using the system files, try using it as a system service.
+
+- user service
+  - `systemctl --user status ovos.service`
+- system service
+  - `systemctl status ovos.service`
+
+## ovos-core
+https://github.com/OpenVoiceOS/ovos-core
+
+This service provides the main instance for OVOS and handles all of the skill loading, and intent processing.
+
+All user queries are handled by the skills service, you can think of it as OVOS's brain
+
+typical systemd command
+
+`systemctl --user status ovos-skills`
+
+[Technical Docs on Skills](https://openvoiceos.github.io/ovos-technical-manual/skills_service/)
+
+## Messagebus
+https://github.com/OpenVoiceOS/ovos-messagebus
+
+C++ version
+
+https://github.com/OpenVoiceOS/ovos-bus_service
```

Comment:
add a note this is beta and has know issues where it sometimes crashes  (mostly with the cli utils)